stubs/peewee/playhouse/flask_utils.pyi

```diff
@@ -0,0 +1,22 @@
+from _typeshed import Unused
+from collections.abc import Container
+
+from flask import Flask
+from peewee import Database, Model, Proxy
+
+class FlaskDB:
+    # Omitting undocumented base_model_class on purpose, use FlaskDB.Model instead
+    database: Database | Proxy
+    def __init__(
+        self,
+        app: Flask | None = None,
+        database: Database | Proxy | None = None,
+        model_class: type[Model] = ...,
+        excluded_routes: Container[str] | None = None,
+    ) -> None: ...
+    def init_app(self, app: Flask) -> None: ...
```

Comment:
Right, so here's a catch. Our type-checking tests won't download flask, only stubtest does because it's added to `METADATA.toml` so that it can successfully resolve imports when running. So mypy and pyright will think the `flask` module (and by extension, the `Flask` class) don't exist.

Normally we add the library as a requirement to the stub, but I think that adding `Flask` as a requirement to `types-peewee` is way overkill.

Next we could instead ignore those errors, as they won't bubble up outside this module (for example, if `Flask` was used as a parent class, then it'd lead to more issues). **But**, pyright curiously doesn't raise any issue here, only mypy. I think it might be because pyright still provides fallbacks stubs for Flask v1.

If that's the case, I'd rather not rely on that behaviour.

So the last solution is to simply keep the `app` parameter untyped.
We can still provide a bit of a type hint to the human developer though, by using a type alias.

```suggestion
from typing import Any
from typing_extensions import TypeAlias
from peewee import Database, Model, Proxy

# Is actually flask.Flask
_Flask: TypeAlias = Any

class FlaskDB:
    # Omitting undocumented base_model_class on purpose, use FlaskDB.Model instead
    database: Database | Proxy
    def __init__(
        self,
        app: _Flask | None = None,
        database: Database | Proxy | None = None,
        model_class: type[Model] = ...,
        excluded_routes: Container[str] | None = None,
    ) -> None: ...
    def init_app(self, app: _Flask) -> None: ...
```